citesphere/src/main/java/edu/asu/diging/citesphere/web/user/concepts/types/EditConceptTypeController.java

```diff
@@ -52,8 +55,16 @@ public String post(Model model, @PathVariable("conceptTypeId") String typeId, Au
             @Validated @ModelAttribute("form") ConceptTypeForm form, BindingResult result,
             RedirectAttributes redirectAttributes) {
         if (result.hasErrors()) {
+            
+            for(ObjectError error : result.getAllErrors()) {
+                    redirectAttributes.addFlashAttribute("show_alert", true);
```

Comment:
Address the codefactor flags for indentation

Reply:
Done